python/pacsign/PKCS11_config.json

```diff
@@ -0,0 +1,30 @@
+{
+   "cryptoki_version": [2, 40],
+   "library_version": [2, 5],
+   "platform-name" : "DCP",
+   "lib_path" : "D:/SoftHSM2/lib/softhsm2-x64.dll",
+   "curve": "secp256r1",
+   "token": {
+      "label": "Scott",
+      "user_password": "intel123",
```

Comment:
@r-rojo I'm not familiar with the PACSign tool's internal config files, but is this a dummy config/password?

Reply:
@buie1 @asgibbons

Reply:
@r-rojo - can you please replace the following fields with dummy values (`foo`, `bar`, etc):

- `lib_path`
- `token.label`
- `token.user_password`